service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java

```diff
@@ -64,9 +68,23 @@ private void configureRouteClaimSubmitForFull() {
         .process(FunctionProcessor.fromFunction(saveToDbService::insertClaim))
         // Use Properties not Headers
         // https://examples.javacodegeeks.com/apache-camel-headers-vs-properties-example/
+        .setProperty("id", simple("${body.recordId}"))
```

Comment:
I would name this to be a bit less generic.  claim-db-id or something like that.